src/main/java/core/basesyntax/service/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private FruitStorageDao storageDao;
+
+    public ReturnOperation(FruitStorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction) {
```

Comment:
It's great to see that you're correctly overriding the handle method from the OperationHandler interface. This ensures that the ReturnOperation class conforms to the expected behavior of an operation handler.